src/fmu/dataio/providers/objectdata/_base.py

```diff
@@ -83,7 +83,7 @@ def __post_init__(self) -> None:
             metadata[usecontent] = getattr(
                 content_model.content_incl_specific, usecontent, None
             )
-
+        metadata["product"] = None
```

Comment:
This is just a placeholder now, functionality to set this will be added in a separate PR